src/Avalonia.Diagnostics/Diagnostics/Models/ConsoleContext.cs

```diff
@@ -0,0 +1,36 @@
+﻿#pragma warning disable IDE1006 // Naming Styles
+
+using Avalonia.Diagnostics.ViewModels;
+
+namespace Avalonia.Diagnostics.Models
+{
+    public class ConsoleContext
```

Comment:
Does this have to be `public` ?

Reply:
That does seem to be the case unfortunately:

![image](https://user-images.githubusercontent.com/1775141/74679014-e66fa780-51bc-11ea-9bd9-aefa63e4e35a.png)

I wonder if there's any way around this?

Reply:
No idea, just trying to minimize public API if possible.